angular/src/app/auth.service.ts

```diff
@@ -68,13 +70,19 @@ export class AuthService {
   }
 
   afterLogin(jwt: string) {
+    this.jwt = jwt;
```

Comment:
This is not called after JWT refresh => logout after some time even if webpage is used.